frontend/src/components/Modal/RiderModalInfo.tsx

```diff
@@ -128,10 +129,28 @@ const RiderModalInfo = ({
           <Label className={styles.label} htmlFor="needs">
             Needs:{' '}
           </Label>
-          <Input
+          <select name="needs" ref={register({ required: true })}>
+            {[
+              'Crutches',
```

Comment:
would be better to update the `Accessibility` enum in types, then replace this array with `Object.values(Accessibility)`